CONTRIBUTING.md

```diff
@@ -24,7 +24,10 @@ For more information, please see STYLEGUIDE.md
 
 # <a name="Attachments"></a>Attachments
 
-GitHub only allows image attachments. Other attachments, for example zipped logs, can be posted in the <a href="http://www.universalmediaserver.com/forum/viewtopic.php?f=14&t=1656">Github attachments thread</a> in the UMS forum.
-Remember to include a link to the attachment in your post here.
+GitHub only allows a few attachment extensions: .png, .gif, .jpg, .docx, .pptx, .xlsx, .txt, .pdf, .zip and .gz.  
```

Comment:
@Sami32 Thit was my suggestion (https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1311#discussion_r122598408), and the reason to list them is the amount of confusion this system causes. I had to look for a while to even figure out what extensions was allowed. A lot of image formats isn't allowed, and the same goes for archives. Even 7zip, which is very popular, isn't allowed - so it's a bit hard for anyone to guess from a general description.